contracts/compute_engines/yield_curve/YieldCurve.cairo

```diff
@@ -48,7 +47,7 @@ func oracle_controller_address_storage() -> (oracle_controller_address : felt):
 end
 
 @storage_var
-func publisher_key_storage() -> (publisher_key : felt):
+func future_spot_pontis_source_key_storage() -> (future_spot_pontis_source_key : felt):
```

Comment:
Need to update the initialize yield curve script

Reply:
https://github.com/42labs/Pontis/pull/74/commits/a70f0cc1194c4301fc34c54f92cfdc3a29bd6076